lib/hammer_cli_foreman_azure_rm.rb

```diff
@@ -1,6 +1,7 @@
 module HammerCLIForemanAzureRM
```

Comment:
I think it would be better if we change the `HammerCLIForemanAzureRM` constant entirely to `HammerCLIForemanAzureRm` in order to avoid any unexpected errors such as may be Unitialized Constant.